thirdeye/thirdeye-pinot/src/main/resources/com/linkedin/thirdeye/dashboard/views/tabs/analysis/analysis-options.ftl

```diff
@@ -0,0 +1,47 @@
+<div class="analysis-card padding-all">
+    <div class="analysis-title bottom-buffer">Trend Analysis
+      <span class="analysis-details">
+        (for <label class="label-medium-semibold">Metric</label> {{metricName}} from <label class="label-medium-semibold">Dataset</label> {{dataset}})
+      </span>
+    </div>
+    <div class="analysis-options">
+      <div class="analysis-options__datepicker">
+      <div class="datepicker-field">
+        <label class="label-medium-semibold">Date Range (Current) </label>
+        <div id="current-range" class="datepicker-range">
+          <span></span><b class="caret"></b>
+        </div>
+      </div>
+
+      <div class="datepicker-field">
+        <label class="label-medium-semibold">Compare To (Baseline)</label>
+        <div id="baseline-range" class="datepicker-range">
+          <span></span> <b class="caret"></b>
+        </div>
+      </div>
+
+      </div>
+      <div class="analysis-options__dropdown">
+        <div class="analysis-options__granularity">
+          <label for="granularity">Granularity </label>
```

Comment:
Please see comment regarding a11y for forms - adding "for" attribute. Same for the rest in this RB.